src/kontrol/__main__.py

```diff
@@ -806,6 +808,13 @@ def _parse_test_version_tuple(value: str) -> tuple[str, int | None]:
         action='store_true',
         help='Flag used by Simbolik to initialise the state of a non test function as if it was a test function.',
     )
+    prove_args.add_argument(
+        '--xml-test-report',
+        dest='xml_test_report',
+        default=False,
+        action='store_true',
+        help='Generate a JUnit xml report',
```

Comment:
One very minor spelling suggestion: 

```suggestion
        help='Generate a JUnit XML report',
```